src/spikeinterface/benchmark/tests/test_benchmark_sorter.py

```diff
@@ -56,10 +65,148 @@ def create_a_study(study_folder):
     # print(study)
 
 
-def test_SorterStudy(setup_module):
+def _create_complex_study(study_folder):
```

Comment:
Will this be run in github CI ? This llokds like enormous!!

Reply:
yeah might be quite large. I'll trim it a bit